backend/graphql/resolvers/courseResolvers.ts

```diff
@@ -28,11 +28,20 @@ const getCourseVisibilityAttributes = (
 ): CourseVisibilityAttributes => {
   switch (role) {
     case "User":
-      return { private: false, published: true };
+      return {
+        includePrivateCourses: false,
+        includeOnlyPublishedModules: true,
```

Comment:
Maybe it could be worth chaning it to `includeDraftModules`? I think that makes more sense, as long as it's clear that draft is the opposite of published.

Reply:
Had this discussion synchronously and both had weak opinions, but basically:
- Idea of "draft" is currently limited to frontend, don't see a compelling argument to bring it into the backend esp. since the backend calls it published and therefore anyone working in the backend (probably) understands it easiest this way
- The double negative is gross but not to the point that it's not understandable
- Can always adjust it as we go, as other devs touch this code

@draymond63 did i miss anything?